### ISSUE TITLE
Cache isn't updating

### ISSUE THREAD

**Issue by RyanSinger** [keybase/client#5540]
`Ryans-MacBook-Pro:~ ryan$ keybase id m1 
▶ INFO Identifying m1
✔ <tracked> public key fingerprint: 88DA 9ED7 6DD2 985C AA91 6EFB 57DB 7AD2 EEAF 6A3E
You last followed m1 on 2017-01-12 13:12:43 PST
✔ <tracked> "weirdorange" on github: https://gist.github.com/0683faba64199d464b04 [cached 2017-01-24 12:15:22 PST]
`

https://keybase.io/m1 shows that he has 2 devices. When I look on my local client, I see only this: 
![image](https://cloud.githubusercontent.com/assets/2665386/22271937/1d97ef58-e24c-11e6-8293-a02f37b1cfa7.png)

How do I force dump the cache so I'll get the updated view?

**Comment by cjb** [comment]
I don't think we render the device count in the GUI yet. What makes you think it's a cache issue?

**Comment by RyanSinger** [comment]
Because I wasn't seeing them on the command line, either?

It might be a UX issue instead :-)

**Comment by RyanSinger** [comment]
BTW, Congratz on the chat launch! I was very excited for it.

**Comment by cjb** [comment]
Yeah, just a website UX that hasn't been made for the local client yet. CC @malgorithms

Thanks re Chat! Not announcing it for real until some more bugs are fixed.
